hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -34,4 +34,53 @@ void HologramDataSourceAvailabilityFetcher::
                                                 &hologram_configs_));
 }
 
+void HologramDataSourceAvailabilityFetcher::UpdateProto(std::string system, 
+    std::time_t time, DataSource data_source, StatusType status) {
```

Comment:
std::time_t time -> int64 timestamp_millis

Reply:
time_t is the structure I need to convert to struct tm though

Reply:
As chatted offline, please use int64, absl::Time, absl::CivilDay etc.